onnx_xla/translations/translate_unsqueeze.cc

```diff
@@ -0,0 +1,42 @@
+#include "onnx_xla/operator_registry.h"
+
+namespace onnx_xla {
+// Translate Unsqueeze by using XLA's Reshape
+onnxStatus translateUnsqueeze(const Node& n,
+                              XlaBuilder& builder,
+                              ValueOpMap& valueToOp) {
+  // Set origShape and axes
+  std::vector<int64> origShape = OperatorRegistry::parseOnnxInputSizes(n, 0);
```

Comment:
unrelated to this PR: `parseOnnxInputSizes` probably shouldn't belong to registry, let's move it out

Reply:
let's change it to int64_t

Reply:
What if I move it and other util functions I've put in OperatorRegistry to a utils file? Also I wanted to keep it int64_t at first but at least two operators use the vector and pass to the builder (which requires int64), so this would incur a copy converting int64_t -> int64. Please advise whether I should change to int64_t and incur a copy, have two functions for both return types, or keep the same?

Reply:
moving to utils files sounds good.
which operators?

Reply:
oh I thought you were saying two functions in xla. copy should be fine as this is the shape which is small (I had really bad experiences of using non-standard c++ types, it caused all kinds of random issues when compiling with different compilers/os/libraries)